spec/ConfigChanges/ConfigFile.spec.js

```diff
@@ -78,14 +78,19 @@ describe('ConfigFile tests', function () {
                 expect(ConfigFile.resolveConfigFilePath('project_dir', 'android', 'strings.xml')).toBe(stringsPath);
             });
 
+            it('should return android values xml file path', function () {
+                const resPath = path.join(projectDir, 'res', 'values', 'colors.xml');
+                expect(ConfigFile.resolveConfigFilePath('project_dir', 'android', path.join('res', 'values', 'colors.xml'))).toBe(resPath);
+            });
+
             it('should return ios config.xml file path', function () {
-                spyOn(ConfigFile, 'getIOSProjectname').and.returnValue('iospath');
+                ConfigFile.__set__('getIOSProjectname', () => 'iospath');
```

Comment:
I think we should avoid using `rewire` when possible and stick with the jasmine API for mocking.

Rewire arguably promotes [bad practices](https://medium.com/decathlondigital/should-i-test-private-methods-direclty-c48f4fa7bb4d) and sticking with Jasmine will make the tests more simple.

I'd like to see `rewire` be completely removed eventually but of course there is a lot of legacy code, and perhaps code that isn't easily unit testable without it (which is a problem with the code in question)